src/tests/test_index_btree.cpp

```diff
@@ -449,6 +455,97 @@ TYPED_TEST(BtreeTest, ThreadedCpFlush) {
     LOGINFO("ThreadedCpFlush test end");
 }
 
+template < typename TestType >
+struct BtreeConcurrentTest : public BtreeTestHelper< TestType > {
+
+    using T = TestType;
+    using K = typename TestType::KeyType;
+    using V = typename TestType::ValueType;
+    class TestIndexServiceCallbacks : public IndexServiceCallbacks {
```

Comment:
Is there a difference in Setup() and TearDown() of BtreeTest vs BtreeConcurrentTest, can we make it same ?
Even the TYPED_TEST(BtreeConcurrentTest, ConcurrentAllOps) can be put in BtreeTestHelper so that both index and membtree can use it.

Reply:
The way that they are called are slightly different for membtree because it does use only iomgr whereas index needs homestore. 
I have plan to expand concurrent. It is better to separate it from non-concurrent UTs

Reply:
Ideally both concurrent and non current UT's should call same set of API's.